src/main/java/hudson/plugins/sloccount/SloccountResult.java

```diff
@@ -25,7 +25,7 @@
 
     private transient SloccountReport report;
 
-    private final AbstractBuild<?,?> owner;
+    private transient final Run<?,?> owner;
```

Comment:
I am not sure this change is enough.
The `owner` field won't be persisted to the disk anymore, and I do not see logic where it gets restored after loading from the disk. Maybe it's not required